app/models/concerns/text_sanitizer.rb

```diff
@@ -37,4 +42,21 @@ def sanitize_category(kw, max_length, strict)
       kw.strip.slice(0, max_length)
     end
   end
+
+  def add_newlines_to_tags(text)
+    text.gsub(/<p>/i, "\n<p>")
+      .gsub(/<\/p>/i, "</p>\n")
+      .gsub(/<div>/i, "\n<div>")
+      .gsub(/<\/div>/i, "</div>\n")
+      .gsub(/<br>/i, "\n<br>")
+      .gsub(/<br\s*\/>/i, "\n<br>")
```

Comment:
Should these check if the tags already have newlines?  Or do we scrub those out before calling this?

Reply:
it dedupes the newlines anyway, so adding another is always fine.

Reply:
Makes good sense!